hedera-mirror-monitor/src/main/java/com/hedera/mirror/monitor/publish/TransactionPublisher.java

```diff
@@ -137,26 +152,34 @@ public void close() {
     }
 
     private Flux<Client> getClients() {
-        List<NodeProperties> validNodes = validateNodes();
+        validateNodes();
 
-        if (validNodes.isEmpty()) {
-            throw new IllegalArgumentException("No valid nodes found");
-        }
-
-        validNodes.forEach(n -> nodeAccountIds.add(AccountId.fromString(n.getAccountId())));
-        Map<String, AccountId> nodeMap = validNodes.stream()
+        log.info("Creating {} connections to {} nodes", publishProperties.getClients(), monitorProperties.getNodes()
+                .size());
+        Map<String, AccountId> nodes = monitorProperties.getNodes().stream()
                 .collect(Collectors.toMap(NodeProperties::getEndpoint, p -> AccountId.fromString(p.getAccountId())));
-        log.info("Creating {} connections to {} nodes", publishProperties.getClients(), validNodes.size());
 
+        NodeValidationProperties validationProperties = monitorProperties.getNodeValidation();
+        if (validationProperties.isEnabled()) {
+
+            nodeValidator = Optional.of(Flux.interval(validationProperties.getFrequency(),
+                            validationProperties.getFrequency())
```

Comment:
nit: don't need separate initial delay and subsequent period, since they are the same
```suggestion
            nodeValidator = Optional.of(Flux.interval(validationProperties.getFrequency())
```

Reply:
Ah, misread the documentation on that, yeah, I'll change it and make sure it still plays well.

Reply:
Done.